examples/advanced/federated-statistics/hierarchical_stats.ipynb

```diff
@@ -0,0 +1,236 @@
+{
+ "cells": [
```

Comment:
suggest moving this notebook file inside the "hierarchical_stats" folder.

Reply:
Done, thanks.